CHANGELOG.md

```diff
@@ -39,6 +39,8 @@ Adds API 500 support to the following HPE OneView resources:
   - oneview_user
   - oneview_volume_template
 
+Adds testing documentation into `TESTING.md`
```

Comment:
This is important to have in the CHANGELOG, but to me it feels somewhat misplaced in this line.
Perhaps put it under a `Notes:` or similar section?
Not a **must** , just a suggestion.